sample_factory/algo/learning/learner.py

```diff
@@ -866,10 +866,13 @@ def _record_summaries(self, train_loop_vars) -> AttrDict:
         stats.act_min = var.mb.actions.min()
         stats.act_max = var.mb.actions.max()
 
-        stats.adv_min = var.mb.advantages.min()
-        stats.adv_max = var.mb.advantages.max()
-        stats.adv_std = var.adv_std
-        stats.adv_mean = var.adv_mean
+        if not self.cfg.with_vtrace:
```

Comment:
Instead of checking for vtrace I suggest we check something like `"adv_mean" in stats` because there can be other reasons advantage stats are not present. Good catch though!!

Reply:
Yah of course, sorry for not noticing the documentation. When I find a moment I'll update the PR.